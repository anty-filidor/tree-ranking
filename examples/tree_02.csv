id;parent_id;color;value;
1;;red;10;
2;1;red;20;
3;1;blue;30;
4;1;green;40;
5;1;green;50;
6;2;pink;60;
7;2;red;70;
8;3;blue;80;
9;3;blue;90;
10;3;red;100;
11;4;green;110;
12;5;green;120;
13;5;green;130;
14;9;blue;140;
15;10;red;150;
16;10;red;160;
17;12;orange;170;
18;14;blue;180;
19;15;orange;190;
20;16;orange;200;
21;17;red;210;
22;21;red;220;
23;18;blue;230;
24;19;orange;240;
25;21;red;250;
26;21;pink;260;
