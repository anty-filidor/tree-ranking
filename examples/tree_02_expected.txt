python color_ranking.py --file /path/tree_02.csv --criteria subtree-count​
[('orange', 3), ('red', 3), ('green', 2), ('pink', 2), ('blue', 1)]

​
python color_ranking.py --file /path/tree_02.csv --criteria subtree-maxdepth​
[('blue', 5), ('red', 3), ('green', 2), ('orange', 2), ('pink', 1)]

​
python color_ranking.py --file /path/tree_02.csv --criteria subtree-average-value​
[('blue', 750), ('red', 396), ('orange', 266), ('green', 225), ('pink', 160)]






